shopping_mall/backend/tests/eval/test_rag_precision.py

```
@@ -0,0 +1,479 @@
+
+    if _svc.chroma_client is None:
+        _rag_skip_reason = "ChromaDB 초기화 실패. uv run python ai/seed_rag.py 를 먼저 실행하세요."
+    else:
+        # 핵심 컬렉션 존재 여부 확인 (faq + return_policy)
+        _faq_col = _svc._get_collection("faq")
+        if _faq_col is None:
+            _rag_skip_reason = "faq 컬렉션 없음. uv run python ai/seed_rag.py 를 먼저 실행하세요."
+        else:
+            _rag_service = _svc
+
+except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)